packages/flutter/lib/src/widgets/_platform_selectable_region_context_menu_web.dart

```diff
@@ -129,6 +129,7 @@ class PlatformSelectableRegionContextMenu extends StatelessWidget {
         'mousedown',
         (web.Event event) {
           final web.MouseEvent mouseEvent = event as web.MouseEvent;
+          mouseEvent.preventDefault();
```

Comment:
Doing this unconditionally will prevent the context menu from showing up. Could you verify please?

I think we should call `preventDefault` inside the `if` below. Do you think that would work?